grr/server/grr_response_server/gui/ui/components/human_readable_size/human_readable_size.ng.html

```diff
@@ -0,0 +1 @@
+<span>{{getHumanSize()}}</span>
```

Comment:
It would be cool to show the exact number of bytes (possibly on hover or behind the human readable size) in a future PR. Analysts might want to copy the exact number of bytes.